src/implementation/Client/GRPCClient/invoker.ts

```diff
@@ -36,7 +36,7 @@ export default class GRPCClientInvoker implements IClientInvoker {
     appId: string,
     methodName: string,
     method: HttpMethod = HttpMethod.GET,
-    data?: object,
```

Comment:
This was the problem @salmankhan-prs

Reply:
You changed the method signature, breaking the tests. I have reverted this back (since we don't have a reason to change it), and only invoker options are added to this signature.